.github/workflows/build.yml

```diff
@@ -4,7 +4,9 @@ on:
   push:
     branches:
       - '**' # target all branches
-
+  pull_request:
+    branches:
+      - master
```

Comment:
Wouldn't the pull requests from external contributors have their separate branch?..

Reply:
This is the target branch